wallets/rn_cli_wallet/src/components/DiscoverList.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+import {ScrollView} from 'react-native';
+
+import projectsData from '../constants/projects-resposne.json';
+import DiscoverListItem from '../components/components/DiscoverListItem';
+
+export default function DiscoverList() {
+  return (
+    <ScrollView
```

Comment:
Totally, FlashList is a must if you want to make sure you have performant lists. I generally even go with Shopify's solution: https://github.com/Shopify/flash-list. Good one. 

The reason I've prefered ScrollView is to use `headerLargeTitle` feature of React Navigation since it requires `contentInsetAdjustmentBehavior="automatic"` property of `ScrollView` and somehow I had some issues while using FlatList with headerLargeTitle, it wasn't quite stable. So I'll double check it. It should support [FlatList](https://reactnavigation.org/docs/native-stack-navigator/#headerlargetitle) too.